src/main/java/edu/cornell/kfs/coa/businessobject/A21SubAccountChange.java

```diff
@@ -0,0 +1,119 @@
+package edu.cornell.kfs.coa.businessobject;
+
+import org.kuali.kfs.coa.businessobject.Account;
+import org.kuali.kfs.coa.businessobject.Chart;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryType;
+import org.kuali.kfs.coa.businessobject.SubAccount;
+import org.kuali.rice.krad.bo.GlobalBusinessObjectDetailBase;
+
+public class A21SubAccountChange extends GlobalBusinessObjectDetailBase {
+	
+	protected String documentNumber;
+	
+    protected String indirectCostRecoveryTypeCode;
+    protected String financialIcrSeriesIdentifier;
+    protected boolean offCampusCode;
+    protected String costShareChartOfAccountCode;
+    protected String costShareSourceAccountNumber;
+    protected String costShareSourceSubAccountNumber;
+    
+    protected Chart costShareChartOfAccount;
+    protected Account costShareAccount;
+    protected SubAccount costShareSourceSubAccount;
+    protected IndirectCostRecoveryType indirectCostRecoveryType;
+	
+	public A21SubAccountChange() {
```

Comment:
technically, we could leave this out and it would be provided by Java by default